Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -46,6 +46,42 @@ const float SHIFT_RIGHT3 = 1.0 / 8.0;
 const float SHIFT_RIGHT2 = 1.0 / 4.0;
 const float SHIFT_RIGHT1 = 1.0 / 2.0;
 
+vec4 computePositionWindowCoordinates(vec4 positionEC, vec2 imageSize, float scale, vec2 direction, vec2 origin, vec2 translate, vec2 pixelOffset, vec3 alignedAxis, float rotation)
+{
+    vec4 positionWC = czm_eyeToWindowCoordinates(positionEC);
+    
+    vec2 halfSize = imageSize * scale * czm_resolutionScale;
+    halfSize *= ((direction * 2.0) - 1.0);
+    
+    positionWC.xy += (origin * abs(halfSize));
+    
+#if defined(ROTATION) || defined(ALIGNED_AXIS)
+    if (!all(equal(alignedAxis, vec3(0.0))) || rotation != 0.0)
+    {
+        float angle = rotation;
+        if (!all(equal(alignedAxis, vec3(0.0))))
+        {
+            vec3 pos = positionEC.xyz + czm_encodedCameraPositionMCHigh + czm_encodedCameraPositionMCLow;
+            vec3 normal = normalize(cross(alignedAxis, pos));
+            vec4 tangent = vec4(normalize(cross(pos, normal)), 0.0);
+            tangent = czm_modelViewProjection * tangent;
+            angle += sign(-tangent.x) * acos(tangent.y / length(tangent.xy));
+        }
+        
+        float cosTheta = cos(angle);
+        float sinTheta = sin(angle);
+        mat2 rotationMatrix = mat2(cosTheta, sinTheta, -sinTheta, cosTheta);
+        halfSize = rotationMatrix * halfSize;
+    }
+#endif
+    
+    positionWC.xy += halfSize;
+    positionWC.xy += translate;
+    positionWC.xy += (pixelOffset * czm_resolutionScale);
+    
+    return positionWC;
+}
+
 void main() 
 {
     // Modifying this shader may also require modifications to Billboard._computeScreenSpacePosition
```

Comment:
Thanks. The shader changes do not need to be reflected there, but it still needs to be updated.